Makefile

```diff
@@ -2910,7 +2930,7 @@ datasets/ptwiki.autolabeled_revisions.10k_2020.json: \
 		datasets/ptwiki.sampled_revisions.10k_2020.json
 	cat $< | \
 	./utility autolabel --host=https://pt.wikipedia.org \
-		--trusted-groups=bot,sysop,bureaucrat \
+		--trusted-groups=bot,sysop,bureaucrat,autoreviewer,rollbacker \
```

Comment:
Not sure why this got picked up by the namefile generator, but it looks right to me.

Reply:
Good to know there is such a generator! I though the make files were created manually...